languages/rust/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -1,8 +1,15 @@
-There are two different types of numbers in Rust:
+## numbers
+
+There are two different types of numbers in Rust.
+
+## integers
 
 - Integers: numbers with no digits behind the decimal separator (whole numbers).
   Integer types can either store only positive numbers (unsigned) or store either positive and negative numbers (signed).
   Examples are -6, 0, 1, 25, 976 and 500000.
+
+## floating-point-numbers
+
 - Floating-point numbers: numbers with zero or more digits behind the decimal separator.
   Examples are -2.4, 0.1, 3.14, 16.984025 and 1024.0.
 
```

Comment:
The text beyond this point, about the name and possible combinations of a numeric type, should not fall under the `floating-point-numbers` header. It should go under the `numbers` header.